oximeter/producer/src/lib.rs

```diff
@@ -481,60 +487,89 @@ mod tests {
         log
     }
 
+    // Re-registration interval for tests.
+    const INTERVAL: Duration = Duration::from_secs(1);
+
+    // Number of requests made to the fake Nexus server.
+    static N_REQUESTS_MADE: AtomicU32 = AtomicU32::new(0);
```

Comment:
Tiny nit, but I'd maybe make this a field of `FakeNexus` (I guess behind an `Arc` so it can also be moved into the handler closure). As written this is fine, but no other test can interact with this correctly, I think, so it seems a little hazardous to leave it as a `static` that they _could_ touch.

Reply:
So, turns out we've written this nice, easy web server called `dropshot`, and that works much better :)